staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -124,6 +124,11 @@ func (s *store) Get(ctx context.Context, key string, resourceVersion string, out
 		if ignoreNotFound {
 			return runtime.SetZeroValue(out)
 		}
+		if len(resourceVersion) > 0 {
+			if rv, err := s.versioner.ParseResourceVersion(resourceVersion); err == nil {
```

Comment:
resourceVersion was not used in the Get call to etcd and had no effect on the response-- I think it's confusing to return it like this. Can we revert this change?